api/schemas/report.json

```diff
@@ -135,14 +134,6 @@
             "type":        "string"
         },
 
-        "kind": {
```

Comment:
That is: I looked in the source code and in the protocol documentation, and I can't find it in either of those places.